### ISSUE TITLE
Missing Log Lines

### ISSUE THREAD

**Issue by mgeitz** [mgeitz/eqalert#61]
The current implementation to read the active log file assumes the log is only written to at most every 0.001 seconds and is never written with more than one line at a time.

During high log traffic, lines are missed.

This is reproducable when receiving a tell with the speak alert set while sitting beside combat between many parties.

Either, `pyinotify` is not fast enough, the updates occur more frequently than every 0.001 seconds, or there is an unknown issue with the log file processor.

**Comment by mgeitz** [comment]
This looks promising - https://stackoverflow.com/questions/3290292/read-from-a-log-file-as-its-being-written-using-python

**Comment by mgeitz** [comment]
This is what is currently producing log entires:
```
    def callback(event):                                                         
    ¦   try:                                                                     
    ¦   ¦   if event.mask == pyinotify.IN_CLOSE_WRITE:                           
    ¦   ¦   ¦   log_q.put(eqa_parser.last_line(char_log))                        
    ¦   except Exception as e:                                                   
    ¦   ¦   eqa_settings.log(                                                    
    ¦   ¦   ¦   "log watch callback: Error on line "                             
    ¦   ¦   ¦   + str(sys.exc_info()[-1].tb_lineno)                              
    ¦   ¦   ¦   + ": "                                                           
    ¦   ¦   ¦   + str(e)                                                         
    ¦   ¦   )  
```

**Comment by mgeitz** [comment]
Considering there is not delay here, it's fair to assume pyinotify is not right for this task or the client write more than one line at a time to the log file.

**Comment by mgeitz** [comment]
Closed by #70
